WordPress/src/test/java/org/wordpress/android/ui/posts/PrepublishingHomeViewModelTest.kt

```diff
@@ -284,6 +284,39 @@ class PrepublishingHomeViewModelTest : BaseUnitTest() {
         assertThat(event).isNotNull
     }
 
+    @Test
+    fun `verify that PUBLISH action is disabled if PostStatus is PRIVATE`() {
+        whenever(editPostRepository.status).thenReturn(PRIVATE)
+
+        viewModel.start(editPostRepository, site)
+
+        val uiState = getHomeUiState(PUBLISH)
+
+        assertThat(uiState?.actionEnabled).isFalse()
+    }
+
+    @Test
+    fun `verify that TAGS action is enabled if PostStatus is PRIVATE`() {
+        whenever(editPostRepository.status).thenReturn(PRIVATE)
+
+        viewModel.start(editPostRepository, site)
+
+        val uiState = getHomeUiState(TAGS)
+
+        assertThat(uiState?.actionEnabled).isTrue()
+    }
+
+    @Test
+    fun `verify that VISIBILITY action is disabled if PostStatus is PRIVATE`() {
```

Comment:
I think the name of this test is supposed say `action is enabled`

Reply:
Resolved here 9f6cd9f